.github/workflows/release.yml

```diff
@@ -0,0 +1,143 @@
+        echo "✅ Latest tag updated to $KO_DOCKER_REPO:latest"
+        
+    - name: Generate changelog
+      id: changelog
+      run: |
+        # Get previous tag for changelog
+        PREVIOUS_TAG=$(git describe --tags --abbrev=0 HEAD^ 2>/dev/null || echo "")
+        
+        if [ -n "$PREVIOUS_TAG" ]; then
+          echo "## Changes since $PREVIOUS_TAG" > CHANGELOG.md
+          git log --oneline $PREVIOUS_TAG..HEAD >> CHANGELOG.md
+        else
+          echo "## Initial Release" > CHANGELOG.md
+          echo "First release of Conforma Verifier Listener" >> CHANGELOG.md
+        fi
+        
+        echo "changelog<<EOF" >> $GITHUB_OUTPUT
+        cat CHANGELOG.md >> $GITHUB_OUTPUT
+        echo "EOF" >> $GITHUB_OUTPUT
+        
+    - name: Create GitHub Release
+      uses: actions/create-release@0cb9c9b65d5d1901c1f53e5e66eaf4afd303e70e # v1.1.4
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ steps.version.outputs.version }}
+        release_name: Release ${{ steps.version.outputs.version }}
+        body: |
+          # Conforma Verifier Listener ${{ steps.version.outputs.version }}
+          
+          ## 🚀 Container Images
+          - **Latest**: `${{ env.QUAY_REPOSITORY }}:${{ steps.version.outputs.version }}`
+          - **Commit-specific**: `${{ env.QUAY_REPOSITORY }}:${{ steps.version.outputs.version }}-${{ steps.version.outputs.short_sha }}`
+          
+          ## 📋 Deployment
+          
+          ### Local Testing
+          ```bash
+          export KO_DOCKER_REPO=quay.io/robnester_rh/knative-service
+          make deploy-staging
+          ```
+          
+          ### Using Release Image
+          Update your kustomization to use the release image:
+          ```yaml
+          images:
+          - name: ko://github.com/conforma/conforma-verifier-listener/cmd/launch-taskrun
+            newName: ${{ env.QUAY_REPOSITORY }}
+            newTag: ${{ steps.version.outputs.version }}
+          ```
+          
+          ## 🔐 Security Features
+          - VSA Attestation support (configurable via `ENABLE_VSA_ATTESTATION`)
+          - SBOM generation included
+          - Supply chain security ready
+          
+          ## 📚 Documentation
+          - [Staging Deployment Guide](STAGING_DEPLOYMENT.md)
+          - [README](README.md)
+          
+          ${{ steps.changelog.outputs.changelog }}
+        draft: false
+        prerelease: false
+
+  notify-infra-deployments:
+    name: Notify infra-deployments
+    runs-on: ubuntu-latest
+    needs: create-release
+    if: success()
+    
+    steps:
+    - name: Repository dispatch to infra-deployments
+      uses: peter-evans/repository-dispatch@ff45666b9427631e3450c54a1bcbee4d9ff4d7c0 # v3.0.0
+      with:
+        token: ${{ secrets.DISPATCH_TOKEN }}
+        repository: redhat-appstudio/infra-deployments
+        event-type: conforma-verifier-listener-release
```

Comment:
Maybe some comments to explain what this is and where to find it?

Reply:
Apologies, I intended to remove this.